src/hardware/developing-an-iox/protocol.md

```diff
@@ -281,6 +281,18 @@ Sent from the IOX to the GO device to inform the GO device of events or status c
 | 0-1 | 0x0004 |
 | 2 | Unused |
 
+#### Information Type 11 IOX Third-Party Tx-Buffer State (GO9+ only)
+The IOX sends a "Stop-sending" message to GO for every Tx Data(0x0B) message it receives when the IOX Tx-queue going to an external third-party device is about to fill-up.
```

Comment:
Should change wording to "IOX can choose to send a"...

Reply:
Changed as suggested.